src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
Hmm, what is the upside of removing this IVF? We have no restrictions against making `internal`s visible to test projects. This change is not needed at all, even if we need `HashCodeCombiner` in the test project (because the Runtime product code doesn't use that class).